team.tf

```
@@ -162,6 +176,12 @@ resource "github_team" "logius-triage" {
   privacy        = "closed"
 }
 
+resource "github_team" "logius-estafettemodel" {
+  name           = "logius-estafettemodel"
+  parent_team_id = github_team.logius.id
```

mist hier een description?
```suggestion
  parent_team_id = github_team.logius.id
  description = "Team dat componenten van Logius door het Estafettemodel haalt"
```